models/airswap/ethereum/airswap_ethereum_schema.yml

```diff
@@ -20,7 +20,7 @@ models:
             - version
             - tx_hash
             - evt_index
-            - trade_id
```

Comment:
line 94-96
`- &trade_id
        name: trade_id
        description: "Ranked record id for the same trade event"
`
also can be removed.

Reply:
thank you!